assets/scss/components/_pen.scss

```diff
@@ -21,12 +21,12 @@
     align-items: center;
     border-bottom: 1px solid $color-alpha-tint-light;
     .Status {
-        padding-left: 10px;
-        margin-left: auto;
+        @include padding-inline('start', 10px);
+        @include margin-inline('start', auto);
     }
     @include mq($until: navCollapse) {
-        padding: 10px 0;
-        margin: 0 15px;
+        padding: 10px 15px;
+        margin: 0;
```

Comment:
Took the opportunity to fix the issue here that we spotted for pen titles on narrower screens.